functions/server/fn_init.sqf

```diff
@@ -6,31 +6,25 @@ GRAD_REPLAY_PLAYBACK_PAUSED = false;
 
 if (!isServer) exitWith {};
 
-params [["_precision", 1]];
-
 // constants
 GRAD_REPLAY_RECORDING_PAUSED = false;
 GRAD_REPLAY_RECORDING_STOPPED = false;
-
 GRAD_REPLAY_SENDING_DELAY = 0.1;
-
-GRAD_REPLAY_SIDES = [west, east, civilian];
-// vehicle setVariable ["GRAD_replay_track", true];
-GRAD_REPLAY_AI_VEHICLES_TRACKED = false; 
-GRAD_REPLAY_AI_ONFOOT_TRACKED = false;
-
-
 GRAD_REPLAY_DATABASE_TEMP = [];
 GRAD_REPLAY_DATABASE = [];
 
-REPLAY_STEPS_PER_TICK = 1;
+// vehicle setVariable ["GRAD_replay_track", true];
+
+GRAD_REPLAY_SIDES = [(missionConfigFile >> "GRAD_Replay"), "trackedSides", [west, east, civilian]] call BIS_fnc_returnConfigEntry;
+GRAD_REPLAY_AI_VEHICLES_TRACKED = [(missionConfigFile >> "GRAD_Replay"), "trackVehicles", false] call BIS_fnc_returnConfigEntry;
```

Comment:
can we make it `trackED` everywhere, currently its a mix between `trackAI` and `trackEDsides`

Reply:
sure